uspace/app/bdsh/cmds/modules/ls/ls.h

```diff
@@ -25,4 +18,15 @@ struct dir_elem_t {
 	vfs_stat_t s;
 };
 
+typedef struct {
+	/* Options set at runtime. */
+	unsigned int recursive;
+	unsigned int sort;
+
+	bool single_column;
+	bool well_formatted;
```

Comment:
Please rename to exact_size and invert the logic to match the -e option (i.e. exact_size is true if -e is provided),